features/config/tasks.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+- hello_world:
+    description: Wish the world hello
+    kwargs:
```

Comment:
kwargs are often JSON in API specs, please provide an example that loads JSON specs in a YAML workflow.

Reply:
Try integrating with the unittest framework we're using:

https://github.com/google-marketing-solutions/bqflow/blob/main/test/tests.py